gandalf.js

```diff
@@ -13,9 +13,9 @@
 
```

Comment:
SCRIPT-I'm impressed with your use of design patterns here, it really enhances the code organization.